pkg/deploy/generator/resources_rp.go

```diff
@@ -37,98 +37,132 @@ func (g *generator) rpManagedIdentity() *arm.Resource {
+	conditions := []interface{}{}
 
-	if !g.production {
-		// override production ARM flag for more open configuration in development
-		rules[0].SecurityRulePropertiesFormat.SourceAddressPrefix = to.StringPtr("*")
-
-		rules = append(rules, mgmtnetwork.SecurityRule{
-			SecurityRulePropertiesFormat: &mgmtnetwork.SecurityRulePropertiesFormat{
-				Protocol:                 mgmtnetwork.SecurityRuleProtocolTCP,
-				SourcePortRange:          to.StringPtr("*"),
-				DestinationPortRange:     to.StringPtr("22"),
-				SourceAddressPrefix:      to.StringPtr("*"),
-				DestinationAddressPrefix: to.StringPtr("*"),
-				Access:                   mgmtnetwork.SecurityRuleAccessAllow,
-				Priority:                 to.Int32Ptr(125),
-				Direction:                mgmtnetwork.SecurityRuleDirectionInbound,
-			},
-			Name: to.StringPtr("ssh_in"),
-		})
+	if g.production {
+		conditions = append(conditions,
+			condition_has_portal_source_ips,
+			condition_no_portal_source_ips,
+		)
 	} else {
-		rules = append(rules,
-			mgmtnetwork.SecurityRule{
+		conditions = append(conditions, nil)
+	}
+
+	for _, cond := range conditions {
+		rules := []mgmtnetwork.SecurityRule{
+			{
 				SecurityRulePropertiesFormat: &mgmtnetwork.SecurityRulePropertiesFormat{
 					Protocol:                 mgmtnetwork.SecurityRuleProtocolTCP,
 					SourcePortRange:          to.StringPtr("*"),
-					DestinationPortRange:     to.StringPtr("445"),
-					SourceAddressPrefix:      to.StringPtr("10.0.8.0/24"),
+					DestinationPortRange:     to.StringPtr("443"),
+					SourceAddressPrefix:      to.StringPtr("AzureResourceManager"),
 					DestinationAddressPrefix: to.StringPtr("*"),
 					Access:                   mgmtnetwork.SecurityRuleAccessAllow,
-					Priority:                 to.Int32Ptr(140),
+					Priority:                 to.Int32Ptr(120),
 					Direction:                mgmtnetwork.SecurityRuleDirectionInbound,
 				},
-				Name: to.StringPtr("dbtoken_in_gateway_445"),
+				Name: to.StringPtr("rp_in_arm"),
 			},
-			mgmtnetwork.SecurityRule{
+			{
 				SecurityRulePropertiesFormat: &mgmtnetwork.SecurityRulePropertiesFormat{
 					Protocol:                 mgmtnetwork.SecurityRuleProtocolTCP,
 					SourcePortRange:          to.StringPtr("*"),
-					DestinationPortRange:     to.StringPtr("8445"),
-					SourceAddressPrefix:      to.StringPtr("10.0.8.0/24"),
+					DestinationPortRange:     to.StringPtr("443"),
+					SourceAddressPrefix:      to.StringPtr("GenevaActions"),
 					DestinationAddressPrefix: to.StringPtr("*"),
 					Access:                   mgmtnetwork.SecurityRuleAccessAllow,
-					Priority:                 to.Int32Ptr(141),
+					Priority:                 to.Int32Ptr(130),
 					Direction:                mgmtnetwork.SecurityRuleDirectionInbound,
 				},
-				Name: to.StringPtr("dbtoken_in_gateway_8445"),
+				Name: to.StringPtr("rp_in_geneva"),
 			},
-			mgmtnetwork.SecurityRule{
+		}
+
+		if !g.production {
```

Comment:
Where is this set?

Reply:
pkg/deploy/assets/generate.go:
```go
//go:generate go run ../../../hack/gendeploy
```

gendeploy runs it both ways, true and false:
```go
func run() error {
	// dev artifacts
	err := generator.New(false).Artifacts()
	if err != nil {
		return err
	}

	// prod artifacts
	return generator.New(true).Artifacts()
}
```